bftcosi/packets.go

```diff
@@ -187,11 +189,9 @@ type responseChan struct {
 }
 
 // Exception represents the exception mechanism used in BFTCosi to indicate a
-// signer did not want to sign.
-// The index is the index of the public key of the cosigner that do not want to
-// sign.
-// The commit is needed in order to be able to
-// correctly verify the signature
+// signer did not want to sign.  The index is the index of the public key of
```

Comment:
can you do a `:%s/   / /g`?

Reply:
This is the result of my auto comment formatting, to keep it fully justified. Vim's `gq` does this by default.